deploy/stack/compose/setup.sh

```diff
@@ -2,10 +2,11 @@
 
 install_docker() {
 	curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-	add-apt-repository "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+	add-apt-repository --yes "deb https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
 	update_apt
 	apt-get install --no-install-recommends containerd.io docker-ce docker-ce-cli docker-compose-plugin
 	gpasswd -a vagrant docker
+	apt-get install docker-compose
```

Comment:
i believe `docker-ce`/`docker-ce-cli` come with the `docker compose` command. is the older `docker-compose` needed?

Reply:
This is just an old, bad habit of mine. You are quite right, I'll fix this!